streampark-common/src/main/java/org/apache/streampark/common/enums/CatalogType.java

```diff
@@ -19,6 +19,9 @@
 
 /** catalog type */
 public enum CatalogType {
+    MYSQL,
+    PGSQL,
+    ORACLE,
```

Comment:
Do MySQL, PostgreSQL, and Oracle need to be defined separately? Is it okay to use JDBC instead of these?

Reply:
MYSQL, PGSQL, ORACLE are only used for the front-end, so that we can see the front-end catalog corresponding database type, in the catalog layer or jdbccatalog